v20.2/enum.md

```diff
@@ -0,0 +1,211 @@
+---
+title: ENUM
+summary: ENUM data types comprise a set of values.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> `ENUM` data types consist of a set of enumerated, static values.
+
+## Syntax
+
+To declare a new `ENUM` data type, use [`CREATE TYPE`](create-type.html):
+
+~~~ sql
+> CREATE TYPE <name> AS ENUM ('<value1>', '<value2>', ...);
+~~~
+
+where `<name>` is the name of the new type, and `<value1>, <value2>, ...` are string literals that make up the type's set of static values.
```

Comment:
maybe note (and give an example) that `name` can be qualified with a db and schema